src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/incentives/IncentivesResource.kt

```diff
@@ -33,15 +32,10 @@ class IncentivesResource(
     telemetryClient.trackEvent("incentives-reports-reconciliation-requested", mapOf("active-prisoners" to activePrisonersCount.toString()))
     log.info("Incentives reconciliation report requested for $activePrisonersCount active prisoners")
 
-    GlobalScope.launch {
-      runCatching {
```

Comment:
no longer need the runCatching since nothing in `incentivesReconciliationService.generateReconciliationReport` will every reasonably thrown anything